### ISSUE TITLE
No notification in logs of Not cancelling active orders when order refresh tolerance is enabled

### ISSUE THREAD

**Issue by rxlxrxsx** [CoinAlpha/hummingbot#2698]
**Describe the bug**

Although the parameter order_refresh_tolerance works as expected, there are no notification in logs/log pane of `Not cancelling active orders ...` when order refresh tolerance is enabled.

**Steps To Reproduce**

1. Run a pure market making bot
2. Set order_refresh_tolerance_pct to 10 to have a higher chance of orders not getting cancelled
3. Notice that when it's time to refresh, it'll either cancel your active orders or it'll stay in the orderbook without any notification that it won't get cancelled due to your current order_refresh_tolerance setting.

**Screenshots**

From screenshot below, I set the `order_refresh_time` to 30 and you'll see that the active order is still there after several minutes without any notification that it won't get cancelled because it's still in the range set on `order_refresh_tolerance_pct` (in my case, 10).

![image](https://user-images.githubusercontent.com/64377055/101435341-2a4bbd00-3947-11eb-8c7f-1801882a12c9.png)

**Release version**

Installation type: Source
Version: dev-0.34.0
Machine: AWS EC2 instance
Operating system: Linux

**Attachments**

Config: [ort.txt](https://github.com/CoinAlpha/hummingbot/files/5656643/ort.txt)
Logs: [ort.log](https://github.com/CoinAlpha/hummingbot/files/5656658/ort.log)

**Comment by CalicaliT** [comment]
Docker build on Mac
- order refresh 10sec
- spreads 1%
- order refresh tolerance 5%
![image](https://user-images.githubusercontent.com/71769411/101954312-ce2eb480-3c36-11eb-94a1-3615f1cbe51e.png)

**Comment by PtrckM** [comment]
Also happens on crypto_com - bot trying to cancel the order during refresh but there is no change on mid price (tolerance 0.0)

![image](https://user-images.githubusercontent.com/54516858/101980031-3c0dc700-3c9d-11eb-913b-038e8e9f2a0d.png)

**Comment by Nullably** [comment]
I commented a few below messages (to no longer log), I did it initially for testing as these messages filled up the log files and make it hard to debug. There are a number of users who complained their log files got very big as well.
So let remove these log message for now, we can put them back on if need be in the next release.
These are the messages removed:
* `Not enough balance for buy (sell) order .... order_amount is adjusted to`
* `Not cancelling active orders since difference between new order prices and current order prices is within...`
